sound/hda/hdac_bus.c

```diff
@@ -17,6 +17,7 @@ static void snd_hdac_bus_process_unsol_events(struct work_struct *work);
 static const struct hdac_bus_ops default_ops = {
 	.command = snd_hdac_bus_send_cmd,
 	.get_response = snd_hdac_bus_get_response,
+	.link_power = snd_hdac_bus_link_power,
```

Comment:
Hmm, might be better to add a check to bus initialize to use this as default if link_power is NULL...

Reply:
Ack, did this, no changes needed to legacy driver now.